python/cuml/decomposition/pca.pyx

```diff
@@ -214,9 +215,13 @@ class PCA(Base):
     iterated_power : int (default = 15)
         Used in Jacobi solver. The more iterations, the more accurate, but
         slower.
-    n_components : int (default = 1)
+    n_components : int / None (default = None)
```

Comment:
```suggestion
    n_components : int (default = None)
```
I think in general, when the default is `None`, we just specify the type of object that could be sent, not needed to add the `or None`